manifests/0000_90_cluster-authentication-operator_01_prometheusrbac.yaml

```diff
@@ -0,0 +1,65 @@
+# Role for accessing metrics exposed by the cluster-authentication-operator
```

Comment:
We should consider making this into a cluster role if it is getting duplicated across all operators (jira for 4.2 seems reasonable).